UPGRADING_TO_OKHTTP_4.md

```diff
@@ -198,3 +201,48 @@ null are uncommon and Kotlin callers may have incorrectly assigned the result to
 [japicmp_gradle]: https://github.com/melix/japicmp-gradle-plugin
 [mockito]: https://site.mockito.org/
 [kotlin_sam]: https://kotlinlang.org/docs/reference/java-interop.html#sam-conversions
+
+
+Companion Imports
```

Comment:
Would it be worth adding a line explaining why `@JvmStatic` isn't being used for better java compatibility here? I could guess (bytecode duplication), but I'm not actually sure from reading this and would wonder why

Reply:
d'oh ignore me, this is for kotlin imports